### ISSUE TITLE
Roadmap to v1.0.0

### ISSUE THREAD

**Issue by PgBiel** [Glistix/glistix#5]
What do we need for a v1 release?

1. Git dependencies (blocked by upstream https://github.com/gleam-lang/gleam/issues/1338)
2. Requirement overriding (blocked by upstream https://github.com/gleam-lang/gleam/issues/2899)
    - We currently work around this using local/Git dependencies, so it's possible this isn't needed for an initial v1 release. Would be nice to have either way.
3. Implement missing functions in stdlib (https://github.com/glistix/stdlib)
4. A bit larger of an ecosystem
    - Library for NixOS modules (which I've been taking a look at) and for `nixpkgs` bindings (and, by extension, nix shell etc.) would be nice
5. Some real-world testing
